src/Streams.cpp

```diff
@@ -36,7 +36,7 @@ void Streams::clearStreams() {
       Stream->stop();
     }
     // Wait for Epics to cool down
-    std::this_thread::sleep_for(std::chrono::milliseconds(1000));
+    std::this_thread::sleep_for(std::chrono::milliseconds(100));
```

Comment:
This wait statement indicates a dangerous and/or bad design. Is it needed at all?

Reply:
Thanks for commenting on this, the change was supposed to be temporary whilst we were testing something. I'll make a quick PR to set it back to the original value.

I suspect it is a work around for something falling over when EPICS objects get cleaned up. I'll add a comment to the EPICS7 ticket so that it gets investigated/removed when we update the EPICS library.